agent/lm_agent/reconciliation.py

```diff
@@ -162,7 +166,21 @@ async def reconcile():
         (_, feature) = product_feature.split(".")
         for config in configs:
             if config.id == config_id:
-                minimum_value = config.features[feature]["total"]
+                # Using the total amount of licenses as the minimum value
+                try:
+                    minimum_value = config.features[feature].get("total")
+                    if not minimum_value:
+                        raise LicenseManagerFeatureConfigurationIncorrect(
```

Comment:
you could use `require_condition` for this:
```python
LicenseManagerFeatureConfigurationIncorrect.require_condition(
    minimum_value,
    f"The configuration for {feature} is incorrect. Please include the total amount of licenses.",
)

Reply:
Thank you, I changed it in the latest commit.